reference/extensions/mapper_selectors.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: page
+title: MapperSelectors Extension
+---
+
+The `MapperSelectors` extension adds `Controller#set_selectors`, which sets selectors in the controller's `identity_map` that ensure the fields returned from `Controller#expanded_fields` (from the [`FieldExpansion` extension](./field_expansion)) are loaded for a given model when `identity_map.load(model)` is called.
+
+This will `select` any required database columns, and `track` any relevant associations.
```

Comment:
Is it true that it will select *only* required database columns? It seems like at least that concept is the reason to use this extension. It allows you to move from SELECT * to SELECT required, right?